### PR TITLE
zh-TW: cr2 Day 2 translations

### PR DESCRIPTION
**Pull request by henrif75**
Professional translations for CR2 content.
Changes are being staged in a separate branch until all zh-TW content is reviewed.

#1463 #684

### CODE REVIEW

File: po/zh-TW.po [hunk 1]
```diff
@@ -5267,6 +5277,9 @@ msgid ""
 "tests. To evaluate a boxed expression, use the deref operator (`*`) to "
 "\"unbox\" it: `eval(*boxed_expr)`."
 msgstr ""
+"這裡的 `Box` 型別是智慧指標，我們會在本課程的後續部分詳細說明。運算式可透過 "
```
Comment (mingyc) [inline po/zh-TW.po:5280]:
```suggestion
"這裡的 `Box` 型別是一種智慧指標，我們會在本課程的後續部分詳細說明。如測試中所示，運算式可被 "
```

File: po/zh-TW.po [hunk 2]
```diff
@@ -5267,6 +5277,9 @@ msgid ""
...
+"`Box::new`「裝箱」，如測試中所示。如要求裝箱運算式的值，請使用 deref 運算子 "
```
Comment (mingyc) [inline po/zh-TW.po:5281]:
```suggestion
"`Box::new`「裝箱」。如要求裝箱運算式的值，請使用 deref 運算子 "
```

File: po/zh-TW.po [hunk 3]
```diff
@@ -5284,53 +5300,58 @@ msgid ""
 "`todo!()` and get the tests to pass one-by-one. You can also skip a test "
 "temporarily with `#[ignore]`:"
 msgstr ""
+"請複製程式碼並貼到 Rust Playground，然後開始實作 `eval`。最終成品應會通過測"
```
Comment (mingyc) [inline po/zh-TW.po:5303]:
```suggestion
"請複製程式碼並貼到 Rust Playground，然後開始實作 `eval`。最終成品應要通過測"
```

File: po/zh-TW.po [hunk 4]
```diff
@@ -5608,64 +5632,71 @@ msgid ""
...
+"不過，下方的 `StderrLogger` 會記錄詳細程度不限的所有訊息。您的任務是編寫 "
+"`VerbosityFilter` 型別，忽略超出詳細程度上限的訊息。"
 
 #: src/methods-and-traits/exercise.md
 msgid ""
 "This is a common pattern: a struct wrapping a trait implementation and "
 "implementing that same trait, adding behavior in the process. What other "
 "kinds of wrappers might be useful in a logging utility?"
 msgstr ""
+"以下是常見模式：結構體包裝一個特徵實作項目，並實作該相同特徵，在程序中加入行"
```
Comment (mingyc) [inline po/zh-TW.po:5654]:
```suggestion
"以下是一種常見模式：在一個結構體中封裝一個特徵的實作，此結構體同時也實作該相同特徵，和在程序中加入行"
```

File: po/zh-TW.po [hunk 5]
```diff
@@ -6822,6 +6910,10 @@ msgid ""
 "selecting the bottom 32 bits of a `u64` with `as u32`, regardless of what "
 "was in the high bits)."
 msgstr ""
+"使用 `as` 進行轉換，可說是採用相對鋒利的一種工具，使用上並不容易；而且要是日"
```
Comment (mingyc) [inline po/zh-TW.po:6913]:
```suggestion
"雖然使用 `as` 進行型別相當簡單，但是非常容易出錯；舉例來說，"
```

File: po/zh-TW.po [hunk 6]
```diff
@@ -6822,6 +6910,10 @@ msgid ""
...
+"後的維護作業改變了所用型別或型別中值的範圍，此工具就可能會成為導致細微錯誤的"
```
Comment (mingyc) [inline po/zh-TW.po:6914]:
```suggestion
"如果往後的維護作業改變了所用型別或型別中值的範圍，這常常就是某些細微錯誤的來源。"
```

File: po/zh-TW.po [hunk 7]
```diff
@@ -6822,6 +6910,10 @@ msgid ""
...
+"來源。只有在意圖用於指明無條件截斷時，我們才建議轉換。舉例來說，如果無論高位"
```
Comment (mingyc) [inline po/zh-TW.po:6915]:
```suggestion
"只有在意圖用於指明無條件截斷時，我們才建議使用型別轉換。舉例來說，如果無論高位"
```

File: po/zh-TW.po [hunk 8]
```diff
@@ -6822,6 +6910,10 @@ msgid ""
...
+"元中的內容為何，您都要透過 `as u32` 選取 `u64` 的底部 32 位元，就可以這麼做。"
```
Comment (mingyc) [inline po/zh-TW.po:6916]:
```suggestion
"元中的內容為何，您都只需要 `u64` 的底部 32 位元，就可以使用 `as u32`。"
```

File: po/zh-TW.po [hunk 9]
```diff
@@ -6830,21 +6922,26 @@ msgid ""
 "casts, `TryFrom` and `TryInto` are available when you want to handle casts "
 "that fit differently from those that don't."
 msgstr ""
+"如果是 `u32` 到 `u64` 這類絕對無誤的轉換，適合先使用 `From` 或 `Into` (而非 "
```
Comment (mingyc) [inline po/zh-TW.po:6925]:
```suggestion
"如果是 `u32` 到 `u64` 這類絕對無誤的型別轉換，適合先使用 `From` 或 `Into` (而非 "
```

**Review by mingyc — CHANGES_REQUESTED** [review]

**Review by mingyc — APPROVED** [review]

**Comment by henrif75** [comment]
Thank you @mingyc!

**Pull request merged**